Singapore/src/main/java/com/infoshareacademy/model/Trip.java

```diff
@@ -1,14 +1,81 @@
 package com.infoshareacademy.model;
 
-import com.infoshareacademy.service.AddUserTrip;
-import com.infoshareacademy.service.TripGenerator;
+import com.infoshareacademy.model.place.Places;
+import java.util.List;
 
-public class Trip extends TripGenerator {
+public class Trip implements Persistent{
 
-    private int id;
+    private long id;
 
-    private TripGenerator tripGenerator;
+    private String name;
 
-    private AddUserTrip addUserTrip;
+    /**
+     * Distance in kilometers
+     */
+    private Double distance;
 
+    /**
+     * Time to spend in hours i.e. 5.5
+     */
+    private Double time_for_trip;
```

Comment:
```suggestion
    private LocalDataTime timeForTrip;
```
nie rozumiem funkcji tego pola.